recipes-kernel/linux/files/iot2050_defconfig_base

```diff
@@ -457,30 +449,18 @@ CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_USB_SUPPORT=y
 CONFIG_USB_VIDEO_CLASS=m
 CONFIG_V4L_PLATFORM_DRIVERS=y
-CONFIG_VIDEO_TI_CAL=m
 CONFIG_V4L_MEM2MEM_DRIVERS=y
+CONFIG_VIDEO_TI_CAL=m
 CONFIG_VIDEO_OV2659=m
 CONFIG_VIDEO_OV5640=m
-# CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
 CONFIG_DRM_I2C_NXP_TDA998X=y
-CONFIG_DRM_PANEL_SIMPLE=y
-CONFIG_DRM_PANEL_OSD_OSD101T2587_53TS=y
 CONFIG_DRM_SII902X=y
 CONFIG_DRM_TOSHIBA_TC358767=y
 CONFIG_DRM_TOSHIBA_TC358768=y
 CONFIG_DRM_TI_TFP410=y
 CONFIG_DRM_TIDSS=y
 CONFIG_DRM_LEGACY=y
-CONFIG_FB_SSD1307=y
-CONFIG_BACKLIGHT_PWM=y
-CONFIG_BACKLIGHT_LP855X=m
-CONFIG_BACKLIGHT_GPIO=y
-CONFIG_BACKLIGHT_LED=y
-CONFIG_FRAMEBUFFER_CONSOLE=y
-CONFIG_LOGO=y
-# CONFIG_LOGO_LINUX_MONO is not set
-# CONFIG_LOGO_LINUX_VGA16 is not set
```

Comment:
These look suspicious, though. Aren't we losing the framebuffer console this way? Did you test?

Reply:
`CONFIG_FB=y` now needed. Display remained black.